docs/site/components/subpage/subpage-directive.js

```diff
@@ -23,6 +23,7 @@ angular
           $http.get('https://api.github.com/repos/GoogleCloudPlatform/gcloud-node/commits?sha=gh-pages&per_page=1')
             .then(function(resp) {
               $scope.lastBuiltDate = moment(resp.data[0].commit.committer.date).format('MMMM Do, YYYY');
+              $scope.lastBuiltDateSm = moment(resp.data[0].commit.committer.date).format('M/D/YY');
```

Comment:
Could we factor this apart a bit?

```javascript
var $scope.lastBuildDate = moment(resp.data[0].commit.committer.date);
```

And then in the template do `{{lastBuildDate.format('...')}}` ?